packages/core/src/interpreter.ts

```diff
@@ -552,13 +583,22 @@ export class Interpreter<
    */
   public send = (
     event: SingleOrArray<Event<TEvent>> | SCXML.Event<TEvent>,
-    payload?: EventData
+    payload?: EventData,
+    sendError = false
```

Comment:
Let's remove this; errors should either be unhandled `error.*` events or "natural" errors thrown from execution of the machine.

Reply:
thank you @davidkpiano I just see your message, in my last commit I was replacing this by checking the type in the event, is that ok or should I do in a different way?